risc0/zkvm/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(extern_types)]
```

Comment:
I think we should avoid using unstable features. Especially on the host side; a stable compiler is supported currently.